From 0b7b7073beb3bcd32e1c6cd50155362fc8bce7df Mon Sep 17 00:00:00 2001
From: Francesco Cosoleto <cosoleto@gmail.com>
Date: Sat, 13 Feb 2010 01:27:09 +0100
Subject: [PATCH] Disable by default automatic name filling with current selected item on mkdir command.

Creating and naming directories with a new original name should be
more frequent case rather than reusing and readapting an already
existing filename.

Signed-off-by: Francesco Cosoleto <cosoleto@gmail.com>
---
 src/cmd.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/cmd.c b/src/cmd.c
index ba7b792..05635b5 100644
--- a/src/cmd.c
+++ b/src/cmd.c
@@ -97,7 +97,7 @@ int output_starts_shell = 0;
 int use_internal_edit = 1;
 
 /* Automatically fills name with current selected item name on mkdir */
-int auto_fill_mkdir_name = 1;
+int auto_fill_mkdir_name = 0;
 
 /* if set, only selection of files is inverted */
 int reverse_files_only = 1;
-- 
1.6.6.1

